addons/point_of_sale/i18n/vi.po

```diff
@@ -681,7 +681,7 @@ msgstr "Tự động mở két tiền mặt."
 #. module: point_of_sale
 #: model:ir.model.fields,field_description:point_of_sale.field_pos_config__iface_available_categ_ids
 msgid "Available PoS Product Categories"
-msgstr "Nhóm sản phẩm khả dụng tại POS"
+msgstr "Nhóm Sản phẩm khả dụng tại POS"
```

Comment:
Trong phần mềm có 3 nhóm sản phẩm:
1. Product Category: để dùng cho nội bộ (kế toán, kho vận, cung ứng)
2. Public Product Category: để dùng để phân loại sản phẩm cho mục đích phân loại, hiển thị ở website
3. PoS Product Category: để dùng để phân loại sản phẩm cho mục đích phân loại, hiển thị ở PoS
```suggestion
msgstr "Nhóm Sản phẩm POS khả dụng"
```
Fyi, @Viindoo/product @Viindoo/project @Viindoo/is